test/amqp-integration-test.py

```diff
@@ -222,14 +223,32 @@ def cleanup():
     else:
         print("\n\nFAILURE %d" % exit_status)
 
-
 exit_status = ExitStatus.OK
 tempdir = tempfile.mkdtemp()
+
+parser = argparse.ArgumentParser(description='Run integration tests')
+parser.add_argument('--no-node', dest="run_node", action="store_false",
```

Comment:
I'm not sure that's easier to script, though. Doing string joins in bash is not the best thing in the world.